app/components/emoji-selector/camera.ts

```diff
@@ -0,0 +1,54 @@
+import Component from "@ember/component";
+import { action } from "@ember/object";
+
+export default class CameraMenu extends Component {
+  args: any;
+
+  @action
+  willRender() {
```

Comment:
```suggestion
  didInsert() {
```

and then add {{did-insert this.didInsert}} to the template